app/models/manageiq/providers/azure/cloud_manager/metrics_capture.rb

```diff
@@ -196,6 +196,9 @@ def get_counters(metrics_conn)
           .list('Microsoft.Compute', 'virtualMachines', target.name, target.resource_group)
           .select { |m| m.name.value.in?(COUNTER_NAMES) }
       end
+    rescue ::Azure::Armrest::RequestTimeoutException # Problem on Azure side
+      _log.warn("Timeout attempting to collect metrics definitions for: #{target.name}/#{target.resource_group}. Skipping.")
```

Comment:
Wanna throw the `target.id` in there too?  Just in case there's two targets with the same `name/resoure_group`.